src/main/java/edu/unc/lib/boxc/migration/cdm/util/EadToCdmHeaderConstants.java

```diff
@@ -0,0 +1,56 @@
+package edu.unc.lib.boxc.migration.cdm.util;
+
+import static edu.unc.lib.boxc.migration.cdm.model.CdmFieldInfo.CDM_ID;
+
+public class EadToCdmHeaderConstants {
```

Comment:
javadoc. I suspect if we turn on code climate this would also want there to be a private constructor since this is a file of static stuff.